pytorch/pytorch_edgeml/graph/rnn.py

```diff
@@ -1117,12 +1136,12 @@ def forward(self, x, brickSize):
         x_bricks = torch.reshape(x_bricks, [oldShape[0], oldShape[1] * oldShape[2], oldShape[3]])
         # x bricks: [brickSize, numBricks * batchSize, featureDim]
         # x_bricks = torch.Tensor(x_bricks)
-        hidd0, out0 = self.rnn0(x_bricks)
+        hidd0 = self.rnn0(x_bricks)
```

Comment:
what happened to output ? Earlier this was 
```
hidd0, out0 = self.rnn0(x_bricks)
```

Reply:
@adityakusupati  I was pointing to this in the mail